src/dbnode/client/config.go

```diff
@@ -117,6 +117,10 @@ type Configuration struct {
 
 	// WriteShardsInitializing sets whether or not to write to nodes that are initializing.
 	WriteShardsInitializing *bool `yaml:"writeShardsInitializing"`
+
+	// ShardsLeavingCountTowardsConsistency sets whether or not to write to leaving shards
```

Comment:
Nit: `whether or not writes to leaving shards count towards consistency, by default they so not`